### PR TITLE
Remove buildbot

### PR DESCRIPTION
**Pull request by lukebakken**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,28 +1,77 @@
...
 compile:
-	mvn clean compile
+	mvn compile
 
 test: unit-test integration-test
 
 test-ts: unit-test integration-test-timeseries
 
-unit-test:
+unit-test: compile
```
Comment (alexmoore) [inline Makefile]: The maven `compile` phase is in the lifecycle of the `test` phase, so we don't need to explicitly call it.  Worse case we compile twice.

Comment (alexmoore) [reply Makefile]: https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html

**Comment by christophermancini** [comment]
👍

File: Makefile [hunk 2]
```diff
@@ -13,16 +26,50 @@ unit-test:
...
+	@echo ''
+	@echo ' Targets:'
+	@echo '-----------------------------------------------------------------'
+	@echo ' all                          - Run everything                   '
+	@echo ' test                         - Run unit & integration tests     '
+	@echo ' unit-test                    - Run unit tests                   '
+	@echo ' integration-test-hll         - Run HLL integration tests        '
+	@echo ' integration-test-timeseries  - Run TS integration tests         '
+	@echo ' integration-test-security    - Run security tests               '
+	@echo '-----------------------------------------------------------------'
```
Comment (alexmoore) [inline Makefile:74]: Should we add `configure-security-certs` here?

Comment (lukebakken) [reply Makefile:74]: https://github.com/basho/riak-java-client/blob/develop/Makefile#L37

Comment (alexmoore) [reply Makefile:74]: Late 👍 then

**Pull request merged**